### ISSUE TITLE
bytecode target does not install stublibs anymore

### ISSUE THREAD

**Issue by olafhering** [ocaml/oasis#117]
Since commit 70c197cc4 ("Make OCamlbuild plugin not assume that dynamic linking is supported.") the shared stublibs are not installed anymore. Reverting that change on top of 2e34867 fixes the issue.

**Comment by ygrek** [comment]
Debian is patching this -> https://github.com/ygrek/extunix/issues/23

**Comment by gildor478** [comment]
Relevant bug (initial bug report):
https://forge.ocamlcore.org/tracker/?func=detail&group_id=54&aid=1251&atid=291

P. Snively solution was to not request for building them and not install them.

If I understand the issue, we should not ask to build them but install them if present.

**Comment by glondu** [comment]
No, they should always be built and installed.

**Comment by glondu** [comment]
OK, I've just read the linked bug report. I wasn't considering cross-compilation.

> If I understand the issue, we should not ask to build them but install them if present.

I agree with "install them if present", but I don't understand what is meant by "ask to build them".

**Comment by gildor478** [comment]
@glondu you are right, this has nothing to do with "ask to build them", this is all about linking with them,

Here is what should be done:
- install the stublibs if they are generated
- don't LINK with them on bytecode platform

In 70c197c, this will mean that we revert the OASISLibrary part, but we keep the MyOcamlbuildBase part.
